src/cmd/root.go

```diff
@@ -359,6 +364,35 @@ func newSubIDFileError() error {
 	return errors.New(errMsg)
 }
 
+func setUpConfiguration() error {
+	logrus.Debug("Setting up configuration")
+
+	configFiles := []string{
+		"/etc/containers/toolbox.conf",
+	}
+
+	userConfigDir, err := os.UserConfigDir()
+	if err == nil {
+		configFiles = append(configFiles, fmt.Sprintf("%s/containers/toolbox.conf", userConfigDir))
+	}
+
+	viper.SetConfigType("toml")
+
+	for _, configFile := range configFiles {
+		viper.SetConfigFile(configFile)
+
+		if err := viper.MergeInConfig(); err != nil {
+			if _, ok := err.(viper.ConfigFileNotFoundError); ok {
```

Comment:
Does it actually work? It doesn't seem like Viper actually throws this error when a file is missing.

Reply:
Yes, it does on my machine.

Reply:
I was wrong :)